docs/ref/models/database-functions.txt

```diff
@@ -680,6 +680,492 @@ that deal with time-parts can be used with ``TimeField``::
     2014-06-16 00:00:00+10:00 2
     2016-01-01 04:00:00+11:00 1
 
+
+.. _math-functions:
+
+Math Functions
+==============
+
+versionadded:: 2.1
+
+.. module:: django.db.models.functions.math
+
+We'll be using the following model in examples of math functions::
+ 
+    class Vector(models.Model):
+        x = models.FloatField()
+        y = models.FloatField()
+
+``Abs``
+=========
```

Comment:
Extra underlines characters. A few titles have this issue.

Reply:
Thanks! Working on it.